ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.12.yaml

```diff
@@ -1,20 +1,3 @@
-base_images:
-  rhel-8-golang-1.20:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-golang-1.20-openshift-4.14
-build_root:
-  image_stream_tag:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-golang-1.20-openshift-4.14
-images:
-- dockerfile_path: Dockerfile
-  from: rhel-8-golang-1.20
-  to: oadp-operator
-- dockerfile_path: build/ci-Dockerfile
-  from: rhel-8-golang-1.20
-  to: test-oadp-operator
```

Comment:
This image is still needed for `operator-e2e-aws` tests.

By proxy, `from: rhel-8-golang-1.20` is also needed, which mean `base_images` is needed.

See L87       from: test-oadp-operator

```
- as: operator-e2e-aws
  steps:
    test:
    - as: e2e
      from: test-oadp-operator
```

This applies to all the configs here.

Reply:
added back base_images and images to op versions files

Reply:
and build_root :grimacing: